modules/decima-ext-model-exporter/src/main/java/com/shade/decima/ui/data/viewer/model/dmf/DMFExporter.java

```diff
@@ -914,24 +915,43 @@ private DMFNode multiMeshResourceToModel(
         @NotNull String resourceName
     ) throws IOException {
         final DMFModelGroup group = new DMFModelGroup(resourceName);
-        final RTTIObject[] parts = object.get("Parts");
-
-        try (ProgressMonitor.Task task = monitor.begin("Exporting MultiMeshResource Parts", parts.length)) {
-            for (int partId = 0; partId < parts.length; partId++) {
-                final RTTIObject part = parts[partId];
-                final RTTIReference meshRef = part.ref("Mesh");
-                final RTTIReference.FollowResult mesh = Objects.requireNonNull(meshRef.follow(project, core));
-                final Transform transform = worldTransformToTransform(part.obj("Transform"));
-                final DMFNode model = toModel(task.split(1), mesh.binary(), mesh.object(), "%s_Part%d".formatted(nameFromReference(meshRef, resourceName), partId));
-                if (model == null) {
-                    continue;
+        if (project.getContainer().getType() == GameType.DSDC) {
+            final RTTIReference[] meshes = object.refs("Meshes");
+            final RTTIObject[] transforms = object.objs("Transforms");
+            try (ProgressMonitor.Task task = monitor.begin("Processing meshes", meshes.length)) {
+                for (int partId = 0; partId < meshes.length; partId++) {
+                    final RTTIReference.FollowResult mesh = Objects.requireNonNull(meshes[partId].follow(project, core));
+                    final Transform transform = worldTransformToTransform(transforms[partId]);
```

Comment:
1. Not `WorldTransform` but `Mat34`
2. The `Transforms` array might be empty, in which case no transformation is applied

See the reference implementation:
https://github.com/ShadelessFox/decima/blob/440835e1cfde9fc0952df9752f2a4d12479ff97d/modules/decima-ext-model-exporter/src/main/java/com/shade/decima/ui/data/viewer/model/isr/SceneSerializer.java#L330